src/event-reduce.ts

```diff
@@ -158,6 +164,31 @@ function canFillResultSetFromLimitBuffer<RxDocumentType>(s: StateResolveFunction
     );
 }
 
+function isBrokenSortedLimitCase<RxDocumentType>(s: StateResolveFunctionInput<RxDocumentType>) {
+    // The issue is specifically with limited, sorted lists having updated items moved to the top.
+    return (
+        !isInsert(s) &&
+        isUpdate(s) &&
+        !isDelete(s) &&
+        hasLimit(s) &&
+        !isFindOne(s) &&
+        !hasSkip(s) &&
```

Comment:
i wonder if this is also happening for queries with a skip in them i.e. query page > 0